### PR TITLE
Fixes a bug on windows

### PR DESCRIPTION
**Pull request by YummyOreo**
Fixes the bug that I tried to fix in https://github.com/ellie/atuin/pull/754.

The bug was that it was not distinguishing between `Release` and `Press`. The "fix" *was* to throw away one  input. But this is pragmatic because:
- Git bash does not send both `Release` and `Press`.
- Cgywin compiles it for linux, runs it as linux, thus the fix wont work.

This fixes it by throwing away any `Release` ones.

### CODE REVIEW

**Comment by YummyOreo** [comment]
For the clippy  warning. Idk if I should split the function up into multiple different ones (I feel like that is outside the scope of this), handle this in the `handle_input` function, or ignore it.

**Review by conradludgate — APPROVED** [review]
This makes a lot more sense. Thanks

**Comment by conradludgate** [comment]
> For the clippy warning. Idk if I should split the function up into multiple different ones (I feel like that is outside the scope of this), handle this in the `handle_input` function, or ignore it.

You can add the `allow(clippy::too_many_lines)` for now. I have a refactor in progress anyway

**Review by conradludgate — APPROVED** [review]

File: src/command/client/search/interactive.rs [hunk 1]
```diff
@@ -88,12 +88,17 @@ impl State {
         None
     }
 
+    #[allow(clippy::to_many_lines)]
```
Comment (conradludgate) [inline src/command/client/search/interactive.rs:91]:
```suggestion
    #[allow(clippy::too_many_lines)]
```

**Review by conradludgate — APPROVED** [review]

**Pull request merged**